whitelister.rb

```diff
@@ -19,22 +19,22 @@ def sg_query
   
   def add_ip(user_ip)
     client.authorize_security_group_ingress({
-	group_id: sg_id,
-	ip_protocol: "-1",
-	cidr_ip: "#{user_ip}/32"
+    group_id: sg_id,
+    ip_protocol: "-1",
+    cidr_ip: "#{user_ip}/32"
     })
   end
   
   def add_tag(user_ip)
-	client.create_tags({
-	  resources: [sg_id],
-	  tags: [
-		{
-		  key: user_ip, 
-		  value: timestamp.to_s
-		} 
-	  ]
-	})
+    client.create_tags({
```

Comment:
Also can you remove the `{` here? It's not needed in Ruby when the hash is the last element in an array or an argument list. And the closing one.